modules/kmm-day1-kernel-module-loading.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-day1-kernel-module-loading_{context}"]
+= Day 1 kernel module loading
+
+Kernel Module Management (KMM) is typically a Day 2 Operator. Kernel modules are loaded only after the complete initialization of a Linux (RCHOS) server. However, in some scenarios the kernel module must be loaded at an earlier stage. Day 1 functionality allows customer to load kernel module during Linux systemd initialization stage, using the Machine Config Operator (MCO).
```

Comment:
Misspelled RHCOS, but also you could use one of the attributes:
:op-system-first: Red Hat Enterprise Linux CoreOS (RHCOS)
:op-system: RHCOS
:op-system-lowercase: rhcos

Also `systemd` needs to be in backticks.

Reply:
Made these changes.